Doc/library/textwrap.rst

```diff
@@ -17,31 +17,41 @@ If you're just wrapping or filling one or two text strings, the convenience
 functions should be good enough; otherwise, you should use an instance of
 :class:`TextWrapper` for efficiency.
 
-.. function:: wrap(text, width=70, **kwargs)
+.. function:: wrap(text, width=70, *, initial_indent="", \
+                   subsequent_indent="", expand_tabs=True, \
+                   replace_whitespace=True, fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, *, max_lines=None, \
+                   **kwargs)
 
    Wraps the single paragraph in *text* (a string) so every line is at most
    *width* characters long.  Returns a list of output lines, without final
-   newlines.
+   newlines.  *width* defaults to ``70``.
 
-   Optional keyword arguments correspond to the instance attributes of
-   :class:`TextWrapper`, documented below.  *width* defaults to ``70``.
+   See :class:`TextWrapper` for documentation on all optional keyword
+   arguments.
 
-   See the :meth:`TextWrapper.wrap` method for additional details on how
-   :func:`wrap` behaves.
 
-
-.. function:: fill(text, width=70, **kwargs)
+.. function:: fill(text, width=70, *, initial_indent="", \
+                   subsequent_indent="", expand_tabs=True, \
+                   replace_whitespace=True, fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, \
+                   *, max_lines=None, **kwargs)
 
    Wraps the single paragraph in *text*, and returns a single string containing the
    wrapped paragraph.  :func:`fill` is shorthand for  ::
 
       "\n".join(wrap(text, ...))
 
-   In particular, :func:`fill` accepts exactly the same keyword arguments as
-   :func:`wrap`.
+   See :class:`TextWrapper` for documentation on all additional optional
+   arguments, which are the same as the optional arguments to :func:`wrap`
+   above.
 
 
-.. function:: shorten(text, width, **kwargs)
+.. function:: shorten(text, width, *, fix_sentence_endings=False, \
```

Comment:
You have two '*' in the signature, first should be removed..

Reply:
(Btw this applies to all 3 sigs)

Reply:
Thank you, I fixed it. 1315486f064aaedd56c5ea4e2ea3a156e8f067d0